test/gtest/ucp/test_ucp_mmap.cc

```diff
@@ -243,6 +243,43 @@ UCS_TEST_P(test_ucp_mmap, alloc) {
     }
 }
 
+UCS_TEST_P(test_ucp_mmap, alloc_cuda) {
+    std::vector<ucs_memory_type_t> mem_types = mem_buffer::supported_mem_types();
+    ucs_status_t status;
+    bool is_dummy;
+
+
+    if (std::find(mem_types.begin(), mem_types.end(),
+                  UCS_MEMORY_TYPE_CUDA) == mem_types.end()) {
+        UCS_TEST_SKIP_R("cuda memory unsupported");
+    }
```

Comment:
why not check all memory types?

Reply:
added for all memtypes but it's possible that tests fail with ROCM/ROCM_MANAGED. I'm not sure if rocm has added alloc support in UCT yet.